tests/test_core.py

```diff
@@ -3983,6 +3982,13 @@ def test_printf(self):
   def test_printf_2(self):
     self.do_run_in_out_file_test('tests', 'core', 'test_printf_2')
 
+  def test_printf_float(self):
+    self.do_run_in_out_file_test('tests', 'printf', 'test_float')
+
```

Comment:
please verify these two new tests pass in non-wasm backend mode (`tests/runner.py ALL.test_printf_float` etc.)